packages/blast-tests/e2e-tests/tests/run-deploy-fund.test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I think testing funds can be part of `compile-run` test. Also, if part of `compile-run`, we are checking for compile succession and we spare some run time by using the same preparation for the test